### ISSUE TITLE
[Feature Request] Autoupdate

### ISSUE THREAD

**Issue by lionkor** [BeamMP/BeamMP-Server#93]
The servers should allow the server owner to update the server, possibly with a command like `server-update`.

**Comment by Starystars67** [comment]
@lionkor did you get this working yesterday?

**Comment by lionkor** [comment]
@Starystars67 yep this works, tested on debian, arch, and tested the windows specifics on windows separately.
Needs testing on windows

**Comment by O1LER** [comment]
How would i go about testing on windows?

**Comment by lionkor** [comment]
Best would be if I just gave you a build that pretends to be outdated... I'll post one here when I get a chance on a Windows PC today or tomorrow.
The alternative is to build it yourself, you only need Visual Studio (not VS Code) with all the C++ stuff that comes with it installed. Then the steps to build it are in the repo, but youll need basic knowledge of how to operate visual studio with CMake instead of the classic solutions. Then you change the version number in include/Common.h from { 3, 2, 0 } to something smaller. Then run the server with the commandline flag `--update`. Just in case someone wants to do that until I post such a version here ;)

**Comment by O1LER** [comment]
So i would checkout dev branch but modify it on its version, so it updates itself. Ill try that tomorrow, for fun and science 😁
